libs/phpseclib/vendor/phpseclib/phpseclib/phpseclib/Crypt/EC/Formats/Signature/IEEE.php

```
@@ -52,15 +52,17 @@ public static function load($sig)
      * Returns a signature in the appropriate format
      *
-     * @param \phpseclib3\Math\BigInteger $r
-     * @param \phpseclib3\Math\BigInteger $s
+     * @param BigInteger $r
+     * @param BigInteger $s
+     * @param string $curve
+     * @param int $length
      * @return string
      */
-    public static function save(BigInteger $r, BigInteger $s)
+    public static function save(BigInteger $r, BigInteger $s, $curve, $length)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$curve'. (Unused Code Rules)

(UnusedFormalParameter)